### DISCUSSION TITLE
笔记导出中的自动同步不可选

### DISCUSSION THREAD

**Discussion by HUGOSAMA1121** [windingwind/zotero-better-notes#1073] [Q&A]
为什么笔记导出中的自动同步不可选呀？
![image](https://github.com/user-attachments/assets/0279f2d4-dc2b-4fc2-834e-cc02711fe010)

**Comment by windingwind** [comment]
<img width="531" alt="image" src="https://github.com/user-attachments/assets/9aea71c6-1c47-4c79-9920-36181873434b">

**Reply by HUGOSAMA1121** [reply]
收到！ 非常感谢
